src/components/landscape-sidebar/geo-description-widget/geo-description-widget-component.jsx

```diff
@@ -30,8 +39,7 @@ const GeoDescriptionWidget = ({ data, loading, error, view }) => {
 
   return (
     <div className={styles.container}>
-      <ShareModal theme={{ shareButton: styles.shareButton }} view={view} viewMode={VIEW_MODE.LANDSCAPE} />
-      <h2 className={styles.title}>{data.title}</h2>
+      <FixedHeader closeSidebar={handleBackClick} title={data.title} view={view} viewMode={VIEW_MODE.LANDSCAPE} autoHeight={true} />
```

Comment:
We might need to add this header to the `error` case to allow users to exit landscape mode also when an error occurs on the geo describer API. We might want to double check with @fannycc what is the desired `title`.